README.md

```diff
@@ -0,0 +1,42 @@
+stack-to-nix
+===========
+
+This is a handy nix function for building stack projects with nix, fully inside the nix sandbox.
+It parses the stack.yaml file and translates the dependencies into a series of fetchUrl, fetchGit calls.
+
+## Example
+
+```nix
+{ pkgs }:
+
+let stackToNix = pkgs.callPackage (fetchTarball https://github.com/serokell/stack-to-nix/archive/master.tar.gz) { };
+in
+stackToNix {
+  # root: the path with stack.yaml. you may want to filter this. for example using nix-gitignore.
+  root = ./.;
+  # shell: get the .env instead of the nix-build derivation. recommended that you do this with shell.nix/default.nix.
+  # see https://github.com/DisciplinaOU/disciplina/blob/master/shell.nix
+  shell = pkgs.lib.inNixShell;
+  # you shouldn't need overrides, but you can ;)
+  overrides = final: previous: with pkgs.haskell.lib; {
+    qtah = overrideCabal previous.qtah (super: {
+      libraryToolDepends = with pkgs.qt5; [ qtbase qttools ];
+    });
+  };
+}
+```
+
+## Problems
+
+This depends on [NixOS/nix#2409](https://github.com/NixOS/nix/pull/2409). It's in our patch set. `nix-env -f https://github.com/serokell/serokell-closure/archive/master.tar.gz -iA nix`
+
+## About Serokell
```

Comment:
I have a push mirror set up [from upstream](https://gitlab.com/transumption/nix-generators/stack-to-nix), please don't include this snippet in this repo.

Reply:
All of it is factually true. Your work on this is also paid for by Serokell. They have the moral right to attribution (but not the legal one). You're welcome to fork.

Reply:
Unfortunately, it is not factually true. I maintain this repo and a few others included into this org, with little to no outside contributions. Since I obviously can't include this snippet in the upstream repo (that is the master copy that automatically pushes to this repo), including it will cause me to abandon this copy.

Reply:
Well, it's up to @jagajaga and @naeospsy to tell us if marketing materials should be included in README files. If you ask me, it feels a little bit out of place. If we have to include marketing materials, I would like the facts to be gotten straight, which is a discussion between contributors to the repo and our marketing department. I think that the source of confusion is that normally Serokell organization hosts repos and holds IP, in this case, the situation might be different, which is—again—matter to clear with Serokell's non-technical personnel. So I don't approve including marketing snip in this patch, but I request clarification from CEO and marketing people as for how to proceed.

Reply:
We've discussed several times, we will include that to *our* repos.